src/Genome.cpp

```diff
@@ -74,25 +80,34 @@ dnaweight_t Genome::pmec() { //Partial MEC
 	return out;
 }
 
+//Questions: what does 60 stand for?
+//FIXME: NOT BEING CALLED AT ALL
 double Genome::mecScore() {
-	double maxMec = this->haplotypes.size() * this->haplotypes[0].size() * 60;
+	double maxMec = this->haplotypes.size() * this->haplotypes[0].size() * (maxCoverageAssumption/2);
```

Comment:
the /2 should be dividing by the "ploidy" (I think it's called that) which for mammals is 2.